docs/testing-platform.md

```diff
@@ -0,0 +1,28 @@
+# Testing AMIs or provisioning in AWS
+
+Sometimes you want to mess around testing something complex (untestable locally e.g. AMI or config changes) in a real env, but you don't want to block the CODE/PROD deployment path.
+
+The simplest way to do that is to spin up a new Auto scaling Group in environment TEST, and deploy your build there.
+
+## deploy your stuff for testing
+1. build your branch of platform/your ami/whatever it is you plan to test
```

Comment:
As you noticed building an artifact on a branch is not straighforward. We could have a teamcity project where you can trigger this manually. Could be useful and not to hard to setup.